maintainers/maintainer-list.nix

```diff
@@ -15833,6 +15833,12 @@
     githubId = 143103;
     name = "Attila Sztupak";
   };
+  szucsitg = {
+    name = "Adam Szucs-Matyas";
+    email = "szucsitg@gmail.com";
+    github = "szucsitg";
+    githubId = 32290917;
+  };
```

Comment:
This change should be in a separate commit with a message like `maintainers: add szucsitg`

Reply:
done